core/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     compile "com.cronutils:cron-utils:9.1.3"
     compile "org.opensearch.client:opensearch-rest-client:${opensearch_version}"
     compile 'com.google.googlejavaformat:google-java-format:1.3'
-    compile "com.amazon.opendistroforelasticsearch:common-utils:${common_utils_version}"
+    compile "org.opensearch:common-utils:${common_utils_version}"
```

Comment:
Undo this please because `common-utils` renaming has not been done yet

Reply:
It appears common-utils main branch has been updated, so we can update the github workflows to look at the main branch for common-utils instead of the `1.0.0-beta1` tag. This way the tests will succeed

Reply:
I have updated the workflows to use the main branch instead at the `1.0.0-beta1` tag. The workflows are passing.